AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -195,4 +197,43 @@ + (NSString *)tableNameForPartition:(NSString *)partition {
   return nil;
 }
 
+- (NSArray<MSPendingOperation *> *)pendingOperationsWithToken:(MSTokenResult *)token {
+  NSString *tableName = [MSDBDocumentStore tableNameForPartition:token.partition];
+  NSString *selectionQuery =
+      [NSString stringWithFormat:@"SELECT * FROM \"%@\" WHERE \"%@\" \"IS NOT NULL\"", tableName, kMSPendingOperationColumnName];
```

Comment:
"IS NOT NULL" shouldn't be in quotes, it looks like this right now:

`SELECT * FROM "TABLE" WHERE "COLUMN_NAME" "IS NOT NULL"`

Reply:
Please leave a comment if you resolve without making a change

Reply:
Ok, blame github... I went to the files page and it only showed changes from 15 commits, not all of them 🤦‍♂️

Reply:
Sorry I did make a change I just didn't push, the changes are now pushed 👍